People/Doctor.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+
+namespace People
+{
+    public class Doctor
+    {
+        private const int PerPatientBonus = 100;
+        private const int QualifiedDoctorPayment = 15000;
+        private const int UnderQualifiedDoctorPayment = 1000;
+        private string _name;
+        private int _age;
+        private int _id;
+        private string _gender;
+        private string _specialty;
+        private int _yearsOfExperience;
+        private bool _hasPhd;
+        private int _salary;
+        private string[] _patientsArray;
+
+        public Doctor(string name, string gender, int age, int id, string specialty, int yearsOfExperience, bool hasPhd)
+        {
+            _name = name;
+            _gender = gender.ToLower();
+            _age = age;
+            _id = id;
+            _specialty = specialty.ToLower();
+            _yearsOfExperience = yearsOfExperience;
+            _hasPhd = hasPhd;
+        }
+
+        public Doctor(string name, string gender, int age, int id, string specialty, int yearsOfExperience, bool hasPhd, string[] patients)
```

Comment:
You can use this to call the other constructor and initialize only _patientsArray here